virtual-desktop/src/app/window-manager/mvd-window-manager/personalization-panel/personalization-panel.component.css

```diff
@@ -0,0 +1,94 @@
+
+
+/*
+  This program and the accompanying materials are
+  made available under the terms of the Eclipse Public License v2.0 which accompanies
+  this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+  
+  SPDX-License-Identifier: EPL-2.0
+  
+  Copyright Contributors to the Zowe Project.
+*/
+
+.personalization-panel {
+  width: 100%;
+  height: 100%;
+  background: #171824f2;
+  position: absolute;
+  z-index: 5000; /* Set to 5000 so panel stacks on top of windows, 
+  but does not take precendence over launchbar UI (6000+ z-index) */
+}
+
+.personalization-panel-item {
+  background-repeat: no-repeat;
+  background-position: center;
+  cursor: pointer;
+  border: none;
+  padding: 70px;
+  -webkit-filter: drop-shadow( 0px 0px 5px #00000063 );
+  filter: drop-shadow( 0px 0px 5px #00000063 );
+}
+
+.panel-item-keyboard{
+  background-image: url('../../../../../../virtual-desktop/src/assets/images/personalization/keyboard.png');
```

Comment:
Is there no better way to do this? It's an extremely long relative path that seems like we will regret eventually.